### PR TITLE
Improve Docs

### PR DESCRIPTION
**Pull request by gregjopa**
Thanks for the awesome theme! This PR fixes some typos and grammar in the docs.

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -550,11 +550,13 @@ When a user requests a page that the server cannot find, a standard *404* error
 
 Hide post meta, actions and comments using front-matter settings:
 
-``` yaml
+``` markdown
```
Comment (LouisBarranqueiro) [inline docs/user.md]: The front matter is in yaml. please revert this change.

**Comment by gregjopa** [comment]
I updated the PR based on your feedback. Let me know if any other changes are needed.

**Comment by LouisBarranqueiro** [comment]
Thanks! 👍
